WaterdogPE/src/main/java/cn/stevei5mc/serverhelper/waterdogpe/commands/base/CommandBase.java

```diff
@@ -13,7 +13,7 @@
  * 下列方法有修改的需要再进行重写，只需重写不在列表之内的方法即可正常使用。<br>
  * The following methods need to be overridden only if modifications are required. Simply override methods not in this list for normal usage.
  *
- * <br><br> onExecute <br> buildCommandOverloads <br> addSubCommand <br> checkPermission <br>
+ * <br>列表 ( List ) : <br>- onExecute <br>- buildCommandOverloads <br>- addSubCommand <br>- checkPermission <br>- getAliasesArray
```

Comment:
建议在注释中明确列出所有需要重写的方法，以便开发者更容易理解和维护。

```suggestion
/**
 * 下列方法有修改的需要再进行重写，只需重写不在列表之内的方法即可正常使用。<br>
 * The following methods need to be overridden only if modifications are required. Simply override methods not in this list for normal usage.
 *
 * <br>列表 ( List ) : <br>- onExecute <br>- buildCommandOverloads <br>- addSubCommand <br>- checkPermission <br>- getAliasesArray
 */